verification/curator-service/src/index.ts

```diff
@@ -1,11 +1,14 @@
 import express from 'express';
+import dotenv from 'dotenv';
 
 // Controllers (route handlers).
 import * as homeController from './controllers/home';
 import * as sourcesController from './controllers/sources';
 
 const app = express();
 
+dotenv.config();
+
 // Express configuration.
 app.set('port', process.env.PORT || 3001);
```

Comment:
should we change that?

Reply:
This line was forwards-compatible. This is still what you'd want using dotenv.

Reply:
Oh cool :)